utils/jwe.ts

```diff
@@ -0,0 +1,55 @@
+import * as E from "fp-ts/Either";
+import * as TE from "fp-ts/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import * as jose from "jose";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { Second } from "@pagopa/ts-commons/lib/units";
+
+/**
+ * JWE Generation
+ */
```

Comment:
Since this PR blocks https://github.com/pagopa/io-web-profile-backend/pull/19, please 

As this PR is blocking https://github.com/pagopa/io-web-profile-backend/pull/19, please bring the changes made there.